### PR TITLE
Remove distance and use points instead

### PR DESCRIPTION
**Pull request by epaminond**

### CODE REVIEW

File: imports/api/actions/methods.js [hunk 1]
```diff
@@ -53,15 +51,13 @@ export const insert = new ValidatedMethod({
...
-        distance,
         day: actionDay,
         createdAt: date
       });
 
       Meteor.users.update(Meteor.user()._id, {
         $set: {
-          'points': userPoints + points,
-          'distance': userDistance + distance
+          'points': userPoints + points
```
Comment (OksentukVlad) [inline imports/api/actions/methods.js]: @epaminond We can use $inc here. Thereby we don't need to get userPoints.

File: imports/api/actions/methods.js [hunk 2]
```diff
@@ -20,10 +20,8 @@ export const insert = new ValidatedMethod({
 
     if (actions[action] || projects[project]) {
       const userPoints = Meteor.user().points;
```
Comment (OksentukVlad) [inline imports/api/actions/methods.js]: Why are we using Meteor.user() and Meteor.userId() in method? Is this code running on client only?

Comment (epaminond) [inline imports/api/actions/methods.js]: Not sure,it was there before. )

Comment (OksentukVlad) [inline imports/api/actions/methods.js]: I looked up in docs. Now we can use Meteor.user() and Meteor.userId() anywhere but publish functions. https://docs.meteor.com/api/accounts.html#Meteor-user

File: imports/api/actions/methods.js (hunk 1 above)
Comment (epaminond) [inline imports/api/actions/methods.js]: Good point, fixed that. :)

**Pull request merged**